public/video-ui/src/components/Video/Display.js

```diff
@@ -3,51 +3,78 @@ import VideoEdit from '../VideoEdit/VideoEdit';
 import VideoAssets from '../VideoAssets/VideoAssets';
 import VideoSelectBar from '../VideoSelectBar/VideoSelectBar';
 import VideoPreview from '../VideoPreview/VideoPreview';
-import VideoAuditTrail from '../VideoAuditTrail/VideoAuditTrail';
 import VideoUsages from '../VideoUsages/VideoUsages';
+import VideoMetaData from '../VideoMetaData/VideoMetaData';
+import YoutubeMetaData from '../YoutubeMetaData/YoutubeMetaData';
+import VideoPoster from '../VideoPoster/VideoPoster';
+import GridImageSelect from '../utils/GridImageSelect';
 
 class VideoDisplay extends React.Component {
 
-  componentWillReceiveProps(nextProps) {
-    if (nextProps.video != this.props.video) {
-      this.props.videoActions.getAudits(nextProps.video.id);
-    }
-  }
+  state = {
+    metadataEditable: false,
+    youtubeEditable: false
+  };
 
   componentWillMount() {
     this.props.videoActions.getVideo(this.props.params.id);
   }
 
   saveVideo = () => {
     this.props.videoActions.saveVideo(this.props.video);
+
+    this.setState({
+      editable: false
+    });
   };
 
   saveAndUpdateVideo = (video) => {
     this.props.videoActions.saveVideo(video);
   };
 
-  updateVideo = (video) => {
-    this.props.videoActions.updateVideo(video);
+  selectVideo = () => {
+    window.parent.postMessage({atomId: this.props.video.id}, '*');
   };
 
-  resetVideo = () => {
-    this.props.videoActions.getVideo(this.props.video.id);
-  };
+  manageEditingState = (value, property) => {
 
-  selectVideo = () => {
-    window.parent.postMessage({atomId: this.props.video.id}, '*');
+    if (property === 'metadata') {
+
+      this.setState({
+        metadataEditable: value
+      });
+
+    } else if (property === 'youtube') {
+
+      this.setState({
+        youtubeEditable: value
+      });
+    }
   };
 
-  enableEditing = () => {
+  disableEditing = () => {
     this.setState({
-      editable: true
+      editable: false
     });
   };
 
   cannotEditStatus = () => {
     return this.props.video.expiryDate <= Date.now();
   };
 
+  renderEditButton = (editable, property) => {
+
+    if (editable) {
+      return (
+        <i className="icon icon__done" onClick={this.manageEditingState.bind(this, false, property)}>done</i>
```

Comment:
"Done editing" perhaps? It'll be a little more explicit for screen reader users.